src/components/AboutVisualizationUnit/AboutVisualizationUnit.js

```diff
@@ -0,0 +1,263 @@
+                            <p
+                                className={cx({
+                                    [classes.noDescription]:
+                                        !data.visualization.displayDescription,
+                                })}
+                            >
+                                {data.visualization.displayDescription
+                                    ? data.visualization.displayDescription
+                                    : i18n.t('No description')}
+                            </p>
+                            <div>
+                                <p>
+                                    <IconShare16 color={colors.grey700} />
+                                    {getSharingSummary(data.visualization)}
+                                </p>
+                                <p>
+                                    <IconClock16 color={colors.grey700} />
+                                    {i18n.t('Last updated {{time}}', {
+                                        time: moment(
+                                            data.visualization.lastUpdated
+                                        ).fromNow(),
+                                    })}
+                                </p>
+                                <p>
+                                    <IconUser16 color={colors.grey700} />
+                                    {i18n.t('Created {{time}} by {{author}}', {
+                                        time: moment(
+                                            data.visualization.created
+                                        ).fromNow(),
+                                        author: data.visualization.createdBy
+                                            .displayName,
+                                    })}
+                                </p>
+                                <p>
+                                    <IconView16 color={colors.grey700} />
+                                    {i18n.t('Viewed {{count}} times', {
+                                        count: data.dataStatistics.views,
+                                        defaultValue: 'Viewed 1 time',
+                                        defaultValue_plural:
+                                            'Viewed {{count}} times',
+                                    })}
+                                </p>
+                            </div>
+                            <div className={classes.subsection}>
+                                <span className={classes.subsectionTitle}>
+                                    {i18n.t('Notifications')}
+                                </span>
+                                {data.visualization.subscribed ? (
+                                    <>
+                                        <p>
+                                            {i18n.t(
+                                                "You're subscribed and getting updates about new interpretations."
```

Comment:
🇬🇧 🤔 ....  "You're subscribed and **are** getting updates about new interpretations."? "You're subscribed and **get** updates about new interpretations."? @cooper-joe

Reply:
For me, it flowed OK.  _You're_  technically applies to the _get_ verb too, so omitting _are_ before _getting_ makes the string shorter.

`You are subscribed and you are getting updates...`
becomes
`You're subscribed and you're getting updates...`
becomes
`You're subscribed and getting updates...`